h/templates/client/annotation.html

```diff
@@ -6,46 +6,36 @@
   <header class="annotation-header">
     <!-- User -->
     <span ng-if="vm.user()">
-      <span>
-        <a class="annotation-user"
-           target="_blank"
-           ng-href="{{::vm.serviceUrl}}u/{{vm.user()}}"
-           >{{vm.username()}}</a>
-      </span>
-
-      <span class="annotation-collapsed-replies">
```

Comment:
Line 136 has that same html. But this one, right here, I'm having a hard time figuring out where its being used.